app/src/app/[lng]/data/review/page.tsx

```diff
@@ -62,30 +55,43 @@ export default function ReviewPage({
 
   const [addUserFile] = api.useAddUserFileMutation();
 
+  const defaultStatus = "pending";
+
+  const [isConfirming, setIsConfirming] = useState<boolean>(false);
+
   const onConfirm = async () => {
-    for (const sector of getAllSectorData) {
-      const formData = new FormData();
-      for (const fileData of sector.files) {
-        const file = appendFileToFormData(
-          fileData.data,
-          `file_XXX.${fileData.fileType}`,
-        );
+    setIsConfirming(true);
+    try {
+      for (const sector of getAllSectorData) {
+        const formData = new FormData();
+        for (const fileData of sector.files) {
+          const file = appendFileToFormData(
+            fileData.data,
+            `${fileData.fileName}`,
+          );
+          formData.append("userId", fileData.userId!);
+          formData.append("fileName", fileData.fileName!);
+          formData.append("sector", fileData.sector!);
+          formData.append("status", defaultStatus);
+          formData.append("fileReference", "");
+          formData.append("url", fileData.url!);
+          formData.append("gpcRefNo", "");
```

Comment:
This should be set to the reference number of the current sector? So either I, II or III.